libs/portal-layout-styles/src/styles/primeng/sass/theme/extensions/_calendar.scss

```diff
@@ -52,16 +52,37 @@
     border-bottom: 0 none;
 
     .p-datepicker-title {
-      margin: 0 auto 0 0;
-      order: 1;
+      order: 2;
+
+      .p-datepicker-year:focus {
+        outline: 1px solid var(--primary-color);
+        outline-offset: 0;
+        box-shadow: none;
+      }
+
+      .p-datepicker-month:focus {
+        outline: 1px solid var(--primary-color);
+        outline-offset: 0;
+        box-shadow: none;
+      }
     }
 
     .p-datepicker-prev {
-      order: 2;
+      order: 1;
+      &:focus {
+        outline: 1px solid var(--primary-color);
+        outline-offset: 0;
+        box-shadow: none;
+      }
     }
 
     .p-datepicker-next {
       order: 3;
+      &:focus {
+        outline: 1px solid var(--primary-color);
+        outline-offset: 0;
+        box-shadow: none;
+      }
```

Comment:
same thing to consider as above comments

Reply:
changed it to $calendarFocusOutline